pkgs/top-level/python-packages.nix

```
@@ -2694,6 +2694,12 @@ in {
 
   ffmpeg-python = callPackage ../development/python-modules/ffmpeg-python { };
 
+  fenics = disabledIf isPy27 (callPackage ../development/libraries/science/math/fenics {
```

you should move disable logic into the nix-expression
```suggestion
  fenics = callPackage ../development/libraries/science/math/fenics {
```